stdlib/multiprocessing/context.pyi

```diff
@@ -201,6 +201,13 @@ if sys.platform != "win32":
         Process: ClassVar[type[ForkServerProcess]]
 
 def _force_start_method(method: str) -> None: ...
-def get_spawning_popen() -> Any | None: ...
-def set_spawning_popen(popen: Any) -> None: ...
+
+if sys.platform != "win32":
+    def get_spawning_popen() -> popen_forkserver.Popen | popen_spawn_posix.Popen | None: ...
```

Comment:
Should I use here `popen_fork.Popen | None`? (`popen_fork.Popen` is parent for `popen_forkserver.Popen` and `popen_spawn_posix.Popen`)
and `Protocol` for returns types "_DupFd" below? (instead of `popen_forkserver._DupFd | popen_spawn_posix._DupFd | resource_sharer.DupFd`)